.github/ISSUE_TEMPLATE/feature-request.yaml

```diff
@@ -0,0 +1,45 @@
+name: "💡 Feature Request"
+description: Request an ideas for this project
+title: "💡 [REQUEST] - <title>"
```

Comment:
We have [bug-report template](https://github.com/euphony-io/euphony-hub-collector/blob/master/.github/ISSUE_TEMPLATE/bug-report.yaml) in ```euphony-hub-collector```.
So, I want to match consistency :)
As you can see, title is:
```yaml
title: "[Bug] <title>"
```

So, for consistency, it would be good:
```suggestion
title: "💡 [Request] <title>"
```